### PR TITLE
Fixed wording in faq_de 'Auffrischungsimpfung' to ' Auffrischimpfung'…

### PR DESCRIPTION
**Pull request by browkin**
Fixed wording in faq_de 'Auffrischungsimpfung' to 'Auffrischimpfung'   Issue #2162

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@browkin / @svengabr 

If you change the FAQ reference to
https://www.coronawarn.app/de/blog/2021-09-22-cwa-version-2-10/ then you need to change the blog article as well.

The article is still headed:
"Version 2.10: Corona-Warn-App ist bereit, schnell über Auffrischungsimpfungen zu informieren" and uses the term "Auffrischungsimpfungen" throughout.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Related also to Blog article
https://www.coronawarn.app/de/blog/2021-09-22-cwa-version-2-10/

FAQ and Blog need to stay consistent.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the hint.

**Comment by dsarkar** [comment]
@MikeMcC399 I think now FAQ and blogs are consistent.

**Review by dsarkar — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@dsarkar 
> I think now FAQ and blogs are consistent.

The one blog article has been fixed. That's good.

There are some other places where the term is used as well, so let me mark those for you too. Stand by.

**Comment by MikeMcC399** [comment]
Locations using "Auffrischungsimpfung"

## Blogs

https://www.coronawarn.app/en/blog/2021-09-08-cwa-version-2-9/
`<center>  </center>`

It's the wrong language in any case. That should be "Booster Vaccination". The text is shown on mouse hover over the image.

## Screenshots

https://www.coronawarn.app/de/screenshots-archive/2-9.html
https://www.coronawarn.app/de/screenshots-archive/2-10.html

You might not want to fix the screenshots as there was a policy decision not to maintain anything except the current version.

**Comment by svengabr** [comment]
I replaced some additional terms

2021-12-08 14_32_49-cuisines-cwa-website - Visual Studio Code

File: blog/2021-09-08-CWA-2-9/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Users who get vaccinated with **Johnson & Johnson’s vaccine** after recovery a
 Furthermore, users who have received a **booster vaccination** can transfer their digital certificate into the Corona-Warn-App. Booster vaccinations are displayed like the other vaccination certificates with "Vaccination n of m".
 
 <br></br>
-<center> <img src="./booster-impfung_v2_en.png" title="Auffrischungsimpfung" style="align: center" width=250> </center>
+<center> <img src="./booster-impfung_v2_en.png" title="Auffrischimpfung" style="align: center" width=250> </center>
```
Comment (MikeMcC399) [inline blog/2021-09-08-CWA-2-9/index.md:61]:
```suggestion
<center> <img src="./booster-impfung_v2_en.png" title="Booster Vaccination" style="align: center" width=250> </center>
```

The original text was in the wrong language.

Comment (dsarkar) [reply blog/2021-09-08-CWA-2-9/index.md]: @MikeMcC399 Thanks.

**Review by MikeMcC399 — APPROVED** [review]
I think we're good to go!

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks....

**Pull request merged**